test/resources/lcmap-landsat.edn

```diff
@@ -27,6 +27,7 @@
  :worker    {:exchange "test.lcmap.landsat.worker.exchange"
              :queue    "test.lcmap.landsat.worker.queue"}
  :search     {:index-url      "http://localhost:9200/tile-specs"
+              :refresh-url    "http://localhost:9200/tile-specs/_refresh"
```

Comment:
Had that same thought... the previous code was actually closer to convention vs config.  I pulled these out in anticipation of two things: 1 - read vs write hosts and 2 - changes to the ES url endpoints in subsequent versions.   Maybe overly cautious on #2.  For #1 we could add a read-host and write-host, remove refresh-url/bulk-api-url/search-api-url if 2 isn't a concern.